<launch>
	<include file="$(find rob_auto_cal)/launch/ueye_launch.launch" />
	<node pkg="rob_auto_cal" name="chess_trans_finder" type="chess_trans_finder" args="0.02393" />
	<include file="$(find rob_auto_cal)/launch/ur10_joint_publish.launch" />
	<node pkg="rob_auto_cal" name="save_joints_node" type="save_joints_node" output="screen" args="/misc/shome/ex09/Thesis/Jointlog" />
</launch>
 
